scripts/install/functions.sh

```diff
@@ -86,6 +86,7 @@ function create_shell_app {
 function add_b2b {
     if [ "${ADD_B2B_LIBS}" = true ] ; then
         ng add --skip-confirmation @spartacus/organization@${SPARTACUS_VERSION} --interactive false
+        ng add --skip-confirmation @spartacus/checkout@${SPARTACUS_VERSION} --interactive false --features="Checkout-Scheduled-Replenishment"
```

Comment:
related to thread for installation script

Reply:
Answered there